rules/falco_rules.yaml

```diff
@@ -3226,7 +3226,7 @@
 
 - macro: known_aks_mount_in_privileged_containers
   condition:
-    (k8s.ns.name = kube-system and container.image.repository = mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi
+    (k8s.ns.name = kube-system and container.image.repository in (mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi,mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi)
```

Comment:
Thank you @kellybirr!

Unrelated to your changes what is the expected behavior, could it be that there is an existing bug and we want the following (I am exaggerating with my formatting for illustration)? WDYT?

```
(

(k8s.ns.name = kube-system and container.image.repository in (mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi, mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi))

or 

(k8s.ns.name = system and container.image.repository = mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver)

)
```

Reply:
You make a good point.  I was unsure if I should call it a bug or a feature.  The expected behavior IMO would be that Falco default rules don't alert on this condition for any built-in AKS CSI driver.  As the rules exist today, it only supports the "azuredisk" driver.

I think your example above would be an improvement over my proposed change.